ansible/roles/base-buster/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Remove apt repo
+  file:
+    path: /etc/apt/sources.list.d/ftp_nl_debian_org_debian.list
+    state: absent
+
+- name: Remove apt repo
+  file:
+    path: /etc/apt/sources.list.d/security_debian_org.list
+    state: absent
+
+- name: Set apt repos
+  template:
+    src: templates/sources.list
+    dest: /etc/apt/sources.list
+    mode: 0644
+    owner: root
+
+- name: Update apt cache and upgrade packages
+  apt:
+    update_cache: yes
+    upgrade: dist
+
+- name: Autoremove
+  apt:
+    autoremove: yes
+
+- name: Installs base packages
+  tags: base-packages
+  apt:
+    cache_valid_time: 86400
+    name:
+      # - prometheus-node-exporter-collectors
+      - bash-completion
+      - byobu
+      - chrony
+      - fail2ban
+      - iotop
+      - manpages
+      - nullmailer
+      - prometheus-node-exporter
+      - rsync
+      - strace
+      - tcpdump
+      - tmux
+
+- name: Clean cache
+  apt:
+    autoclean: yes
```

Comment:
Do you think we should be adding in here also the creation of the accounts, like what dom0 is doing?

Reply:
The bootstrap is quite different from dom0, maybe we can deploy new nodes with these new roles over the next months before starting to backport stuff (due to the risk of breaking live nodes).

Reply:
So currently to deploy this, you would still have to run the dom0 bootstrap playbook on it?

Reply:
Updated to create users